src/util/std_expr.cpp

```diff
@@ -223,3 +224,34 @@ const exprt &object_descriptor_exprt::root_object() const
 
   return *p;
 }
+
+void member_exprt::validate(
+  const exprt &expr,
+  const namespacet &ns,
+  const validation_modet vm)
```

Comment:
Avoid abbreviations

Reply:
Every other `validate` method is doing the same thing, so I'll leave this as-is for now (change them en masse if we want to)